### PR TITLE
using dw instead of cw

### PR DESCRIPTION
**Pull request by rdaly525**
Depends on https://github.com/rdaly525/coreir/pull/759

### CODE REVIEW

**Comment by Kuree** [comment]
You need to add `DW_fp_addsub.v` as well.

**Comment by rdaly525** [comment]
@Kuree Added the appropriate stubs and added copying addsub

File: tests/test_micro.py [hunk 1]
```diff
@@ -100,7 +103,7 @@ def rtl_tester(test_op, data0=None, data1=None, bit0=None, bit1=None, bit2=None,
                                include_verilog_libraries=libs,
                                skip_compile=True)
     else:
-        libs = ["CW_fp_mult.v", "CW_fp_add.v"]
+        libs = ["DW_fp_mult.v", "DW_fp_add.v"]
```
Comment (Kuree) [inline tests/test_micro.py]: You need to add `DW_fp_addsub.v`

**Comment by rdaly525** [comment]
@Kuree, can you check the CAD_ENV var in this branch (test_pe.py)? it looks like it is not getting set on appropriatly and therefore is not running the fp_mult tests on buildkite

**Comment by rdaly525** [comment]
Got the goahead from Keyi

**Pull request merged**